packages/teams-js/src/private/nestedAppAuth/nestedAppAuthBridge.ts

```diff
@@ -0,0 +1,301 @@
+import { v4 as generateUUID } from 'uuid';
+
+/**
+ * Interface representing a request structure.
```

Comment:
Will/do we have any public documentation(at teamsjs doc level) on how this script can be used? @baljesingh @vikramtha

Reply:
Not yet, but we’ll add it soon. @vikramtha , could you please take care of it?